src/engraving/rendering/dev/paint.cpp

```diff
@@ -122,6 +122,11 @@ void Paint::paintScore(Painter* painter, Score* score, const IScoreRenderer::Pai
                 painter->translate(-pageRect.topLeft());
             }
 
+            if (painter->hasClipping() && drawRect.bottom() > pageRect.bottom()) {
```

Comment:
Actually, I think checking only the bottom is not really enough. Elements can fall off the page to the left or right side just as well as on the bottom side (for example, a too-long text element on the last measure of a system may extend beyond the right margin)

Reply:
sure, I couldn't find a case of it, but happy to add those checks.

Reply:
ok, now checks if the drawRect extends outside the pageRect on any side, and if so uses the *whole* page rectangle as the clip rectangle, which works as expected.  In principle it should be possible to do an intersection to work out the minimum clipping rectangle, but it seems to cause too much glitches around the edge.